### PR TITLE
feat: add github actions project in kittehub

### PR DESCRIPTION
**Pull request by pashafateev**
refs: eng-1166

### CODE REVIEW

File: github_actions/README.md [hunk 1]
```diff
@@ -0,0 +1,82 @@
+
```
Comment (daabr) [inline github_actions/README.md]:
```suggestion
```

File: github_actions/README.md [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+# GitHub Workflow Orchestration
+
+This AutoKitteh project automates the process of handling GitHub workflows that interact across multiple repositories, 
+triggering actions in specific repositories based on the completion of workflows in other repositories.
```
Comment (daabr) [inline github_actions/README.md:4]: While this is technically correct, it feels somewhat vague. Maybe describe what this project does specifically, and/or maybe add diagrams like in https://github.com/autokitteh/kittehub/tree/main/task_chain that show what cross-repo/fan-out/fan-in/or-trigger mean? 

Anyway, up to you, and you can do that in a separate PR.

File: github_actions/program.py [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+It defines triggers that automatically start workflows in specific repositories when workflows in other
+repositories are completed.
+
+This program supports several types of triggers:
+1. Cross-repo trigger: Initiates a workflow in repository B when a workflow in repository A completes.
+2. Fan-out trigger: Initiates workflows in repositories B and C upon the completion of a workflow in repository A.
+3. OR trigger: Initiates a workflow in repository C if a workflow in either repository A or B completes.
+4. Fan-in trigger: Initiates a workflow in repository C only when workflows in both repositories A and B complete.
+"""
+
```
Comment (daabr) [inline github_actions/program.py:11]:
```suggestion
```

**Review by daabr — APPROVED** [review]

File: github_actions/README.md (hunk 2 above)
Comment (pashafateev) [reply github_actions/README.md:4]: Yeah that's a good idea and I'll do that in a separate PR
